.rubocop.yml

```diff
@@ -17,6 +17,9 @@ Layout/ArgumentAlignment:
 Layout/EmptyLineAfterGuardClause:
   Enabled: false
 
+AllCops:
+  TargetRubyVersion: '2.7'
```

Comment:
The documentation states that that it can be derived from the gemspec. See https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version

> If a TargetRubyVersion is not specified in your config, then RuboCop will check your project for a series of other files where the Ruby version may be specified already. The files that will be checked are (in this order): *.gemspec, .ruby-version, .tool-versions, and Gemfile.lock.

So I'd prefer to drop this.